docs/next/guides/sidebar.js

```diff
@@ -1,189 +1,189 @@
-  'guides/getting-started/introduction',
-  'guides/getting-started/demo',
-  'guides/getting-started/installation',
-  'guides/getting-started/binding-to-data',
-  'guides/getting-started/saving-data',
-  'guides/getting-started/setting-options',
-  'guides/getting-started/grid-size',
-  'guides/getting-started/events-and-hooks',
-  'guides/getting-started/license-key',
+  { link: 'guides/getting-started/introduction' },
+  { link: 'guides/getting-started/demo' },
+  { link: 'guides/getting-started/installation' },
+  { link: 'guides/getting-started/binding-to-data' },
+  { link: 'guides/getting-started/saving-data' },
+  { link: 'guides/getting-started/setting-options' },
+  { link: 'guides/getting-started/grid-size' },
+  { link: 'guides/getting-started/events-and-hooks' },
+  { link: 'guides/getting-started/license-key' },
 ];
 
 const integrateWithReactItems = [
-  'guides/integrate-with-react/react-installation',
-  'guides/integrate-with-react/react-simple-example',
-  'guides/integrate-with-react/react-modules',
-  'guides/integrate-with-react/react-hot-column',
-  'guides/integrate-with-react/react-setting-up-a-language',
-  'guides/integrate-with-react/react-custom-context-menu-example',
-  'guides/integrate-with-react/react-custom-editor-example',
-  'guides/integrate-with-react/react-custom-renderer-example',
-  'guides/integrate-with-react/react-language-change-example',
-  'guides/integrate-with-react/react-redux-example',
-  'guides/integrate-with-react/react-hot-reference',
+  { link: 'guides/integrate-with-react/react-installation' },
+  { link: 'guides/integrate-with-react/react-simple-example' },
+  { link: 'guides/integrate-with-react/react-modules' },
+  { link: 'guides/integrate-with-react/react-hot-column' },
+  { link: 'guides/integrate-with-react/react-setting-up-a-language' },
+  { link: 'guides/integrate-with-react/react-custom-context-menu-example' },
+  { link: 'guides/integrate-with-react/react-custom-editor-example' },
+  { link: 'guides/integrate-with-react/react-custom-renderer-example' },
+  { link: 'guides/integrate-with-react/react-language-change-example' },
+  { link: 'guides/integrate-with-react/react-redux-example' },
+  { link: 'guides/integrate-with-react/react-hot-reference' },
 ];
 
 const integrateWithAngularItems = [
-  'guides/integrate-with-angular/angular-installation',
-  'guides/integrate-with-angular/angular-simple-example',
-  'guides/integrate-with-angular/angular-modules',
-  'guides/integrate-with-angular/angular-custom-id',
-  'guides/integrate-with-angular/angular-setting-up-a-language',
-  'guides/integrate-with-angular/angular-custom-context-menu-example',
-  'guides/integrate-with-angular/angular-custom-editor-example',
-  'guides/integrate-with-angular/angular-custom-renderer-example',
-  'guides/integrate-with-angular/angular-language-change-example',
-  'guides/integrate-with-angular/angular-hot-reference',
+  { link: 'guides/integrate-with-angular/angular-installation' },
```

Comment:
As an example, can you skip the Angular, Vue 2 and Vue 3 sections in the JavaScript build of the docs?

Reply:
Do you mean `{ link: 'guides/integrate-with-angular/angular-installation' },` page? GitHub shows the suggestion for 28th line.

Reply:
I just added the comment on this line to create a thread. I mean the following:

> As a part of this task, remove "Integrate with Angular", "Integrate with Vue 2", "Integrate with Vue 3" from the docs for React, and remove "Integrate with React" from the docs for JavaScript

Also added to the project brief for clarity